ui/jobs/components/drk.ts

```diff
@@ -20,24 +20,24 @@ export const setup = (bars: Bars): void => {
 
   bars.onJobDetailUpdate((jobDetail: JobDetail['DRK']) => {
     const blood = jobDetail.blood;
-    if (bloodBox.innerText === blood.toString())
-      return;
-    bloodBox.innerText = blood.toString();
-    const p = bloodBox.parentNode;
-    if (blood < 50) {
-      p.classList.add('low');
-      p.classList.remove('mid');
-    } else if (blood < 90) {
-      p.classList.remove('low');
-      p.classList.add('mid');
-    } else {
-      p.classList.remove('low');
-      p.classList.remove('mid');
+    if (bloodBox.innerText !== blood.toString()) {
+      bloodBox.innerText = blood.toString();
+      const p = bloodBox.parentNode;
+      if (blood < 50) {
+        p.classList.add('low');
+        p.classList.remove('mid');
+      } else if (blood < 90) {
+        p.classList.remove('low');
+        p.classList.add('mid');
+      } else {
+        p.classList.remove('low');
+        p.classList.remove('mid');
+      }
     }
 
     const oldSeconds = parseFloat(darksideBox.duration ?? '0') - parseFloat(darksideBox.elapsed);
     const seconds = jobDetail.darksideMilliseconds / 1000.0;
-    if (!darksideBox.duration ?? seconds > oldSeconds) {
+    if (!darksideBox.duration || seconds > oldSeconds) {
```

Comment:
Whoops, I think this was a bad suggestion from me.  <_<

Reply:
yes, blame to him (

Reply:
I recall there are 3 or 4 suggestions like this

Reply:
I double checked and I think the other `??` suggestions are ok.